quinn-proto/src/connection/streams/state.rs

```diff
@@ -240,8 +287,12 @@ impl StreamsState {
 
         // Stopped streams become closed instantly on FIN, so check whether we need to clean up
         if closed {
-            self.recv.remove(&id);
-            self.stream_freed(id, StreamHalf::Recv);
+            let recv = self
+                .recv
+                .remove(&id)
+                .flatten()
+                .expect("must have recv when closing");
```

Comment:
Feel free to use a plain `unwrap` when it's provably impossible to reach.

Reply:
done